src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,9 @@
-<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
 <html>
 <head>
-    <title>My team</title>
+    <title>Taxi service</title>
 </head>
 <body>
-<h1>Hello, mates!</h1>
+<h1>Operation successful!</h1>
 </body>
 </html>
```

Comment:
you don't need to change index.jsp) though i think if you keep it like you did, it won't be fatal.